### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by VioletaScvortova**

### CODE REVIEW

**Review by IvanCeban — COMMENTED** [review]
Gitignore file is missing

**Comment by VioletaScvortova** [comment]
> 

Added!

File: README.md [hunk 1]
```diff
@@ -1 +1,25 @@
...
+## Полезные ссылки
+
+Чтобы узнать больше о синтаксисе Markdown, вы можете обратиться к следующим ссылкам:
+
+- [Официальная страница Markdown](https://daringfireball.net/projects/markdown/)
+- [Основной синтаксис Markdown](https://www.markdownguide.org/basic-syntax/)
+
+Кроме того, вы можете найти более подробный список функций синтаксиса Markdown по адресу:
+- [Полное руководство по синтаксису Markdown](https://www.markdownguide.org/extended-syntax/)
+
```
Comment (IvanCeban) [inline README.md:24]: not ok

File: README.md [hunk 2]
```diff
@@ -1 +1,25 @@
...
+Этот проект является примером использования синтаксиса Markdown.
+
+## Описание
+
+*Markdown* это простой и легкий в освоении язык форматирования текста, часто используемый в проектной документации и веб-страницах. Этот файл README написан в Markdown.
+
+В этом проекте я включил ссылку на официальную веб-страницу Markdown и показал, как форматировать текст, используя различные функции, такие как выделение полужирным шрифтом и создание списков.
+
+## Как использовать этот проект
+
```
Comment (IvanCeban) [inline README.md:12]: optimise

**Review by IvanCeban — APPROVED** [review]

**Pull request merged**